test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -2376,10 +2405,11 @@ public void testGetDocContentListEmptyDefaultView() throws Exception {
             + "Custom List/AllItems.aspx\"", "DefaultViewUrl=\"/\""))        
         .register(SITES_SITECOLLECTION_LISTS_CUSTOMLIST_F_CONTENT_EXCHANGE)
         .register(SITES_SITECOLLECTION_S_CONTENT_EXCHANGE)
-        .register(new URLSegmentsExchange(
-              "http://localhost:1/sites/SiteCollection/Lists/Custom List",
-               true, null, null, "{6F33949A-B3FF-4B0C-BA99-93CB518AC2C0}",
-               null));
+        .register(new URLSegmentsExchange.Builder().setStrURL(
+              "http://localhost:1/sites/SiteCollection/Lists/Custom List")
+               .setGetURLSegmentsResult(true)
```

Comment:
Wrong indent here and following, should be lined up under "i" in ".register".

Reply:
Done